README.md

```diff
@@ -325,6 +325,7 @@ _Sit back, relax, watch, and learn._
 - [vCloudInfo](https://www.youtube.com/vCloudInfo) - Publishes videos based on his home and GitHub repository.
 - [digiblurDIY](https://www.youtube.com/channel/UC5ZdPKE2ckcBhljTc2R_qNA) - Tutorials on hardware projects and Tasmota automations.
 - [Intermit.Tech](https://www.youtube.com/channel/UCv7UOhZ2XuPwm9SN5oJsCjA) - Tutorials & reviews: Camera's, Home Networking, ESP8266 boards, Node-RED.
+- [BeardedTinker](https://www.youtube.com/channel/UCuqokNoK8ZFNQdXxvlE129g) - Tutorials on HomeAssistant 3d printing.
```

Comment:
We don't mention Home Assistant in descriptions, see: <https://github.com/frenck/awesome-home-assistant/blob/master/CONTRIBUTING.md#adding-a-new-link>

`3d printing` -> `3D printing`